<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>importblock</class>
 <widget class="QWidget" name="importblock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>67</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>导入Simulink模型</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>文件名称：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="m_pFileNameEdit"/>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pImportblockButton">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>加载文件</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>模块名称：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="m_pBlockNameEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
